spec/features/create_ride_spec.rb

```diff
@@ -16,4 +18,10 @@
     click_on "Create Ride"
     expect(page).to have_content origin
   end
+
+  scenario "with invalid data" do
+    fill_in "Origin", with: origin
+    click_on "Create Ride"
+    expect(page).to have_selector('#new_ride')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.